projects/views.py

```diff
@@ -70,13 +70,28 @@ def project_member_or_admin_or_superuser(user, project, project_user):
     return False
 
 
-def get_user_project_role_scopes(keycloak_client, user, project):
+def get_user_project_role_scopes(keycloak_client, username, project):
     role, scopes = keycloak_client.get_user_project_role_scopes(
-        user.username, project.chargeCode
+        username, get_charge_code(project)
     )
     return role, scopes
 
 
+def manage_membership_in_scope(scopes):
+    return "manage-membership" in scopes
+
+
+def manage_project_in_scope(scopes):
+    return "manage" in scopes
+
+
+def get_user_permissions(keycloak_client, username, project):
+    role, scopes = get_user_project_role_scopes(keycloak_client, username, project)
+    if not scopes:
```

Comment:
I pushed a patch last night. could you rebase? and I think we can get rid of this check.